docs/reference/api.rst

```diff
@@ -29,4 +29,4 @@ client only sent partial data to update an entity.
         // ...
     }
 
-The call need to be done before the ``bind`` function as you cannot manipulate the form after the binding.
+The call needs to be done before the ``bind`` function as you cannot manipulate the form after the binding.
```

Comment:
Please comply with the sf docs standards:

> [Each line should break approximately after the first word that crosses the 72nd character (so most lines end up being 72-78 characters);](http://symfony.com/doc/current/contributing/documentation/standards.html)

Reply:
sure :-)

Reply:
Ping?